tests/suite/resources_utils.py

```diff
@@ -750,6 +780,21 @@ def get_ingress_nginx_template_conf(v1: CoreV1Api, ingress_namespace, ingress_na
     return get_file_contents(v1, file_path, pod_name, pod_namespace)
 
 
+def get_ingress_nginx_template_ts_conf(v1: CoreV1Api, resource_namespace, resource_name, ingress_name, ingress_namespace) -> str:
+    """
+    Get contents of /etc/nginx/conf.d/{namespace}-{ingress_name}.conf in the pod.
```

Comment:
the doc doesn't match the names used in the function

Reply:
fixed